### DISCUSSION TITLE
db2 supported is for Linux, Unix , Windows - any plans for db2 in IBM i?

### DISCUSSION THREAD

**Discussion by jsanguin** [sequelize/sequelize#14863] [Help & Questions]
I know Sequelize would be of great help in our modernization effort. I just saw that db2 is supported. I don't think it works for db2 on the IBM i. Are there any plans to have it working there ? Thanks

**Comment by WikiRik** [comment]
In the v7 alphas we have added support for Db2 for IBM i; https://sequelize.org/docs/v7/other-topics/dialect-specific-things/#db2-for-ibm-i
